server/main.py

```diff
@@ -49,6 +48,13 @@
 GCS_BUCKET = app.config['GCS_BUCKET']
 _MAX_SEARCH_RESULTS = 1000
 
+RANKING_STATS = {
```

Comment:
Please add a descriptive comment for this const.

Reply:
Done